platform/vs/docker-sonic-vs/Dockerfile.j2

```diff
@@ -24,10 +24,6 @@ RUN apt-get install -y net-tools \
                        python-ply \
                        libqt5core5a \
                        libqt5network5 \
-                       libboost-program-options1.71.0 \
-                       libboost-serialization1.71.0 \
```

Comment:
Hi @saiarcot895 , why removed libboost package?

Reply:
https://github.com/sonic-net/sonic-swss-common/blob/e3d9024602847d5bb41e14f153099431ef35f291/debian/control#L5
swss-common still depends on libboost1.71.

Reply:
Boost 1.71 is not available in Bullseye, Bullseye has Boost 1.74. Because of this (and other differences), the pipelines for swss-common, swss, and sairedis were bound to fail. In addition, the base layer of this container has changed; it's now based on swss-layer-bullseye (which will contain both swss and swsscommon).

As for the pipeline breakages, I've already opened sonic-net/sonic-swss-common#716 to fix it for swss-common. Once that is fixed, I should be able to fix it for sairedis (sonic-net/sonic-sairedis#1225) and swss (PR to be opened).